contrib/cirrus/packer/fedora_packaging.sh

```diff
@@ -60,82 +63,107 @@ INSTALL_PACKAGES=(\
     gcc
     git
     glib2-devel
+    glibc-devel
     glibc-static
     gnupg
     go-md2man
     golang
+    gpgme
     gpgme-devel
+    grubby
+    hostname
     iproute
     iptables
     jq
+    krb5-workstation
+    libassuan
     libassuan-devel
+    libblkid-devel
     libcap-devel
+    libffi-devel
+    libgpg-error-devel
+    libguestfs-tools
```

Comment:
can we add parallel here? it's used for running bats jobs in parallel in newer versions of cri-o

Reply:
We can, but I really recommend against running tests in parallel.  Sometimes it makes sense and is safe, but often it makes debugging incredibly difficult with very little time-savings.  Lastly, our Fedora and Ubuntu setups need to be maintained fairly close, package-wise.

For example, I had a lot of problems running bats in parallel on Ubuntu for Buildah due to unsolvable bugs.  It wasted a lot of Dan and my time debugging, where ultimately I just had to add a condition to skip parallel testing on Ubuntu.  Complicating the setup, and arguably making it less reliable and harder to maintain.

Is it REALLY necessary for conmon, with substantial time-savings (or some other huge benefit)?

Reply:
well, cri-o currently runs the tests in parallel all the time. It is in the cri-o code, not conmon. We'd have to carry a patch to stop cri-o from doing it. The tests are well designed to run in parallel and have been for a long time. If there is an issue there, it would have to be dealt with in cri-o as well (probably by me anyway), so I don't see the risk in adding it for conmon

Reply:
Oh it's in cri-o okay, and I bet the time-savings is quite significant.  I'm okay with adding it since you'll be the one debugging :grin:   If it occurs to you during your cri-o travels, I would suggest adding a parallelism mechanism based on an env. var.  Those are easy to override in most CI systems, in case it's needed for debugging.

Here's how it's done[ in containers/storage ](https://github.com/containers/storage/blob/master/tests/test_runner.bash#L21-L24).  Having the ability to set `JOBS=1` saved me wasting lots of debugging time on Ubuntu (as essplained in the comment).  Otherwise, there's a reasonable default set based on the number of CPUs in the VM.

Reply:
...okay I added 'parallel' to both Ubuntu and Fedora VM images.